OSVR-Unity/Assets/OSVRUnity/src/VRHead.cs

```diff
@@ -63,12 +62,12 @@ void Start()
             {
                 Init();
                 CatalogEyes();
-                _distortionEffect = GetComponent<OsvrDistortion>();
+/*
```

Comment:
I didn't know if the `viewMode == mono` thing was important, given Unity's somewhat unique case of having the editor, the play mode in editor, and standalone mode. If it were just me, I'd remove it, but I didn't want to step on toes.